packages/base/core/src/svg/icons/concept/enterprise-file-storage_concept.svg

```diff
@@ -0,0 +1,2 @@
+<svg id="Calque_1" data-name="Calque 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><defs><style>.cls-1 { fill: #113f6d; }
```

Comment:
**thought:** Both `id=` attributes and `data-name` attribute seems useless. Isn't it?

Reply:
You are right, we used the SVG that was provided